src/parsec.rs

```diff
@@ -31,24 +33,58 @@ pub type Parsec = inner::Parsec<chain::NetworkEvent, FullId>;
 pub type Request = inner::Request<chain::NetworkEvent, id::PublicId>;
 pub type Response = inner::Response<chain::NetworkEvent, id::PublicId>;
 
+// TODO: These two needs to be determined from better data.
+// For now: 640kb ought to be enough for anybody!
+const PARSEC_SIZE_LIMIT: u64 = 640_000;
+// The maximum number of handled requests/responses
+const PARSEC_EVENT_LIMIT: u64 = 1000;
```

Comment:
Do we need both an event limit and a size, why not use one or the other (Maybe more the size).

Reply:
yeah, same doubt. And I will prefer just use size.

Reply:
In the latest update I've remove the event limit and only kept the size limit